alpaca_trade_api/stream2.py

```diff
@@ -158,15 +166,31 @@ def __init__(
             key_id=None,
             secret_key=None,
             base_url=None,
-            data_url=None):
+            data_url=None,
+            data_stream=None):
         _key_id, _secret_key, _ = get_credentials(key_id, secret_key)
         _base_url = base_url or get_base_url()
         _data_url = data_url or get_data_url()
+        if data_stream is not None:
+            if data_stream in ('alpacadatav1', 'polygon'):
+                _data_stream = data_stream
+            else:
+                raise ValueError('invalid data_stream name {}'.format(
+                    data_stream))
+        else:
+            _data_stream = 'alpacadatav1'
+        self._data_stream = _data_stream
 
         self.trading_ws = _StreamConn(_key_id, _secret_key, _base_url)
-        self.data_ws = _StreamConn(_key_id, _secret_key, _data_url)
-        self.polygon = polygon.StreamConn(
-            _key_id + '-staging' if 'staging' in _base_url else _key_id)
+
+        if self._data_stream == 'polygon':
+            self.data_ws = polygon.StreamConn(
+                _key_id + '-staging' if 'staging' in _base_url else _key_id)
+            self._data_prefixes = (('Q.', 'T.', 'A.', 'AM.'))
+        else:
+            self.data_ws = _StreamConn(_key_id, _secret_key, _data_url)
+            self._data_prefixes = (
+                ('Q.', 'T.', 'AM.', 'polyfeed/', 'alpacadatav1/'))
```

Comment:
why both(polyfeed and alpacadatav1)? it's RC.. you don't need to keep supporting it.
also - now that it's an argument we don't need the prefix anymore. just use the same as polygon.

Reply:
sure... thought just to keep the compatiblity for short. will remove it.